test/sagemaker_tests/huggingface/inference/integration/__init__.py

```diff
@@ -28,3 +32,37 @@
 DEFAULT_TIMEOUT = 20
 
 RESOURCE_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "resources"))
+
+
+class NoLogStreamFoundError(Exception):
+    pass
+
+
+class SageMakerEndpointFailure(Exception):
+    pass
+
+
+def dump_logs_from_cloudwatch(e):
+    """
+    Function to dump logs from cloudwatch during error handling
+    """
+    endpoint_regex = re.compile(r"Error hosting endpoint ((\w|-)+):")
+    endpoint_match = endpoint_regex.search(str(e))
+    if endpoint_match:
+        logs_client = boto3.client('logs', region_name='us-west-2')
```

Comment:
Can we declare the region in a variable or pass as a function parameter? The function need to be tweaked a bit when we integrate the multi region capability for TensorFlow.